spateo/plotting/static/pairwise.py

```diff
@@ -0,0 +1,185 @@
+"""
+Plots for pairwise comparison analyses between groups of samples and across different conditions (e.g. biological
+conditions or using different clustering parameters).
+
+These functions build on the Label class- see spateo.utils.Label for documentation
+"""
+from typing import List, Tuple, Union
+
+import matplotlib as mpl
+import matplotlib.patheffects as PathEffects
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy
+from anndata import AnnData
+from matplotlib.collections import PolyCollection
+from matplotlib.ticker import StrMethodFormatter
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+from ...utils import Label, interlabel_connections
```

Comment:
Looks like the sorting of the imports is not correct? did the isort work properly?

Reply:
From a glance, the sorting seems consistent with other files in the repository- I think any **import** statements are ordered before any **from** statements, but within each of the import blocks and for blocks all statements are placed in alphabetical order.